cli/src/main/java/io/kestra/cli/AbstractApiCommand.java

```diff
@@ -90,7 +90,7 @@ protected String apiUri(String path) {
             throw new IllegalArgumentException("'path' must be non-null and start with '/'");
         }
 
-        return tenantId == null ? "/api/v1" + path : "/api/v1/" + tenantId + path;
+        return tenantId == null ? "/api/v1/main" + path : "/api/v1/" + tenantId + path;
```

Comment:
Use the MAIN_TENANT var from the TenantService

```suggestion
        return tenantId == null ? "/api/v1/MAIN_TENANT" + path : "/api/v1/" + tenantId + path;
```